lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -61,6 +61,13 @@ sub _ofac_xml {
         pattern  => '%m/%d/%Y',
         on_error => 'croak',
     );
+
+    # Pattern to look for: mm/dd/yyy
+    my $pattern = '((0[1-9]|1[012])[/]0[1-9]|[12][0-9]|3[01])[/]((19|20)\d\d)';
+    if ($ref->{publshInformation}{Publish_Date} !~ m/$pattern/) {
+        die 'Date does not match the pattern of %m/%d/%Y';
```

Comment:
would be better to die with `Date does not match the pattern of mm/dd/yyyy` for better readability.

Reply:
Done. Made changes for `_hmt_csv` as well.